helm/hpcc/templates/_helpers.tpl

```diff
@@ -1339,7 +1339,7 @@ Pass in dict with root, me and instances defined
 {{- $stubInstanceResources := .stubResources | default .root.Values.global.stubInstanceResources | default dict }}
 {{- $milliCPUText := $stubInstanceResources.cpu | default "200m" }}
 {{- $milliCPUs := int (include "hpcc.k8sCPUStringToMilliCPU" $milliCPUText) }}
-{{- $memoryText := $stubInstanceResources.memory | default "50Mi" }}
+{{- $memoryText := $stubInstanceResources.memory | default "500Mi" }}
```

Comment:
previous value was 200Mi ?

Reply:
Just looking at that memory again.  Previously for eclagent it was 200 * max active, same for thor stub.
So actually 800 for eaclagent stub and 400 for thor by default.  Should I just use 400 for all?